src/components/TodosContext.tsx

```diff
@@ -0,0 +1,85 @@
+import React, {
+  Dispatch,
+  SetStateAction, useMemo, useState,
+} from 'react';
+import { Status, TodoType } from '../types/todo';
+
+export const TodosContext = React.createContext({
+  todos: [] as TodoType[],
+  setTodos: {} as Dispatch<SetStateAction<TodoType[]>>,
+  selectedStatus: Status.all,
+  setSelectedStatus: {} as Dispatch<SetStateAction<Status>>,
+  title: '' as string,
+  setTitle: {} as Dispatch<SetStateAction<string>>,
+  visibleTodos: [] as TodoType[],
+});
+
```

Comment:
just create the context type before the context
```suggestion

type TodosContextType = {
   ....
} 

export const TodosContext = React.createContext<TodosContextType>({
  todos: [] as TodoType[],
  setTodos: {} as Dispatch<SetStateAction<TodoType[]>>,
  selectedStatus: Status.all,
  setSelectedStatus: {} as Dispatch<SetStateAction<Status>>,
  title: '' as string,
  setTitle: {} as Dispatch<SetStateAction<string>>,
  visibleTodos: [] as TodoType[],
});

```
and remove all assertions here